roles/dotnet/handlers/verify.yml

```diff
@@ -0,0 +1,11 @@
+- name: Wait for AppDynamics.Agent.Coordiantor to start
+  win_wait_for_process:
+    process_name_pattern: 'AppDynamics.Coordinator'
+    state: present
+    timeout: 60
```

Comment:
The Coordinator takes a bit longer on older servers.. can you increase it to 120 secs and parameterise it in default variables?

Reply:
Ok, done.

Reply:
Increased to 120, made it tunable via var.